classes/class-p4-search.php

```diff
@@ -737,7 +737,7 @@ public function add_load_more( $args = null ) {
 		 * View the Search page template.
 		 */
 		public function view() {
-			Timber::render( $this->templates, $this->context );
+			Timber::render( $this->templates, $this->context, self::DEFAULT_CACHE_TTL );
```

Comment:
Wanted to add this as well since we do not have FPC for search results, but I had not tested it...

I think we could also add the 4th arg to set cache_mode as `CACHE_OBJECT`.
If we don't then it tries to `get_transient( $key )` which does check if an external object cache is being used in which case leads again to checking redis cache instead of going for sql. So, I think we can directly go for checking redis db by setting the 4th arg. What you think?

Reply:
Ahhh great, I wasn't sure about that, I'll add it and test.